middlewares/team.middleware.js

```diff
@@ -119,6 +120,65 @@ async function ensureSpace(req, res, next) {
     return next();
 }
 
+async function updateTeam(req, res, next) {
+    const team = await Services.Team.updateOne(req.body.team._id, req.body.teamDetails);
```

Comment:
What's the difference between `team._id` and `teamDetails`?

Reply:
Why are there two objects here?

Reply:
A mistake on how I was doing it previously. Changed